addons/l10n_hk/models/res_bank.py

```diff
@@ -12,7 +12,7 @@ class ResPartnerBank(models.Model):
 
     country_code = fields.Char(related='partner_id.country_code', string="Country Code")
     l10n_hk_fps_type = fields.Selection([('id', "FPS ID"), ('mobile', "Mobile Number"), ('email', "Email Address")], string='FPS Type')
-    l10n_hk_fps_identifier = fields.Char(string="FPS ID/Mobile Number/Email Address")
+    l10n_hk_fps_identifier = fields.Char(string="FPS Identifier")
```

Comment:
I would add a help giving more information if needed. Could be confirmed/validated by a FC or PO if any are related to the task.
```suggestion
    l10n_hk_fps_identifier = fields.Char(string="FPS Identifier", help="An FPS ID, mobile number or email address.")
```